tests/model/test_ocrd_page.py

```diff
@@ -139,6 +139,22 @@ def test_simple_types(self):
         self.assertEqual(reg.get_type(), 'page-number')
         self.assertTrue(isinstance(reg.get_type(), str))
 
+    def test_orderedgroup_export_order(self):
+        """
+        See https://github.com/OCR-D/core/issues/475
+        """
+        with open('tests/model/TEMP1_Gutachten2-2.xml', 'r') as f:
+            pcgts = parseString(f.read().encode('utf8'), silence=True)
+            og = pcgts.get_Page().get_ReadingOrder().get_OrderedGroup()
+            rris = og.get_RegionRefIndexed()
```

Comment:
Wait, this only gives you the `RegionRefIndexed` (with a gap in the index range), but not the `OrderedGroupIndexed`.

Reply:
Yep, the example I used was too simplistic.